src/common.ts

```diff
@@ -18,6 +18,7 @@ export interface RequestOptions {
   jwt?: string;
   headers?: Headers;
   body?: any;
+  qs?: object;
```

Comment:
I was using `qs` because `qs` is what the `request` module uses and so having it written as something else at a higher level would only make things less clear I reckon.